js/app.js

```diff
@@ -81,20 +68,98 @@ TextApp.prototype.setTheme = function() {
   this.editor_.setTheme(theme);
 };
 
+/**
+ * Remove the editor so it can be reinitialized.
+ * @param editor The DOM element containing the editor.
+ */
+TextApp.prototype.removeEditor = function(editor) {
```

Comment:
name the arg editorElement to distinguish it from the editor object? I found it a bit confusing

Reply:
Done!